.travis.yml

```diff
@@ -1,22 +1,34 @@
 language: python
 python: 3.5
 
-# Scripts to install dependencies
-install:
-    - make env
-    - make test-env
+# Run in ubuntu-trusty environment
+dist: trusty
+sudo: required
 
-# Environmental variables
-env:
-    # Marks build successful regardless of whether allowed-failures finished
-    fast_finish: true
+# Cache .vagga images to speedup sequential builds
+cache:
+  timeout: 21600 # 6 hours
+  pip: true
+  directories:
+   - .vagga/.roots
+   - .vagga/.mnt
+   - .vagga/.transient
 
-    # Will be set always
-    global:
-        - BLUEBERRYPY_CONFIG='{}'
+# Scripts to install dependencies
+# These are mostly honestly borrowed from https://github.com/tailhook/vagga/tree/master/.travis.yml
+install:
+  - 'echo ubuntu-mirror: http://mirrors.us.kernel.org/ubuntu/ > ~/.vagga.yaml'
+  - echo "$(id -un):100000:65536" | sudo tee /etc/subuid | sudo tee /etc/subgid
+  - sudo apt-get install uidmap -y
+  - curl http://files.zerogw.com/vagga/vagga-install-testing.sh | sh
+  - vagga _build test
+  # I did not find a way to run pre-commit inside vagga
+  - pip install pre-commit
 
 # Run tests
-script: make test-envs
+script:
+  - vagga test --with-coverage --cover-package=GDGUkraine --cover-xml
+  - pre-commit run --all-files
```

Comment:
The problem is `pre-commit` 
1. writes logs to ~/.pre-commit, which is RO in vagga container
2. Tries to create its own virtualenv in ~/.pre-commit which is RO in vagga container

I'm not sure if it is a good idea to create a persistent volume in test container for pre-commit needs